guides/common/modules/con_troubleshooting-postgresql.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Troubleshooting_Postgresql_{context}"]
+= Troubleshooting PostgreSQL
+
+If the PostgreSQL 12 module has already been enabled, perform a module reset:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module reset postgresql
+----
+
+If a database was previously created using PostgreSQL 12, perform an upgrade:
+
+. Enable the DNF modules:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable {dnf-modules}
```

Comment:
This only applies to `dnf` I assume; becaues both "yum" on EL7 (unsupported anyway) and "apt-get" on Debian/Ubuntu does not support "modules enable ...".

Reply:
Agreed and thanks. I assumed `package-manager` attribute as global. :+1: